build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -162,6 +162,27 @@ disableTXChecksumOffload: {{ .Values.disableTXChecksumOffload }}
 # also adjust MTU to accommodate for tunnel encapsulation overhead (if applicable).
 defaultMTU: {{ .Values.defaultMTU }}
 
+# featureRateLimit defines the rate-limiting of different features. All numbers in
+# this struct stand for the rate as packets per second(pps) and the
+# burst/queueSize will be automatically set to 2 times of it.
+# When burst/queue is full, new packets will be dropped.
+featureRateLimit:
+{{- with .Values.featureRateLimit }}
+  # dnsInterception is the rate limiting of sending packets to Antrea-agent for FQDN
+  # DNS interception.
+  dnsInterception: {{ .dnsInterception }}
+  # traceflow is the rate limiting of sending packets to Antrea-agent for traceflow.
+  traceflow: {{ .traceflow }}
+  # networkPolicy is the rate limiting of sending packets to Antrea-agent for
+  # NetworkPolicy logging and reject.
+  networkPolicy: {{ .networkPolicy }}
+  # igmp is the rate limiting of sending packets to Antrea-agent for IGMP.
+  igmp: {{ .igmp }}
+  # svcReject is the rate limiting of sending packets to Antrea-agent for Service
+  # packets not matching any Endpoints.
+  svcReject: {{ .svcReject }}
+{{- end }}
```

Comment:
Sorry, I'm hesitating if it's worth to be so feature specific which might make it a bit harder to understand the configurations, and wondering if there could be a real case that users want to configure different rate-limiter for different features. https://github.com/antrea-io/antrea/issues/5358 seems not asking a per-feature configuration.
Do you think it makes sense to just have one configuration for all packet-in cases like `packetInRate`? @jianjuns @antoninbas @GraysonWu

Reply:
Just to double-check, for one configuration, you mean those feature shares the same packet-in rate-limit config instead of the rate-limit bandwidth right?

Reply:
I feel it works better if we can define an overall rate limit that applies to all packets to antrea-agent. If that is not possible, as shared limit (applies to individual feature separately) also works for me.

Reply:
@GraysonWu Yes, I mean all features share the same limit.
@jianjuns I think it's technically possible to have an overall rate limit but would require many changes as currently both userspace queue and kernel meter have a separate queue/meter for each feature. Besides, I feel it could perhaps cause problems like DNS resolution slowness because the packetins have different real time requirements, DNS interception should be processed as fast as possible because it could affect DNS resolution time while there is no rush to process packetins for NetworkPolicyLogging. Keeping separate meter and queue for each feature could avoid load of each feature from affecting each other.

Reply:
Cool. Pushed a version with all features sharing limit config.